#!/usr/bin/python python
"""
This python configuration file
maps all the motor on the mob
robot and gives them name that
are easy to interface with
"""

# Head
cam  = 15
head = 14

# right arm
right_shoulder = 29
right_elbow    = 30
right_hand     = 31

# left arm
left_shoulder  = 28
left_elbow     = 27
left_hand      = 26

# right leg
right_hip   = 16
right_thigh = 17
right_knee  = 18
right_ankle = 19
right_foot  = 20

# left leg
left_hip   = 21
left_thigh = 22
left_knee  = 23
left_ankle = 24
left_foot  = 25

